src/algebra/algebra/basic.lean

```diff
@@ -911,6 +911,16 @@ instance aut : group (A₁ ≃ₐ[R] A₁) :=
   inv := symm,
   mul_left_inv := λ ϕ, by { ext, exact symm_apply_apply ϕ a } }
 
+@[simp] lemma mul_apply (e₁ e₂ : A₁ ≃ₐ[R] A₁) (x : A₁) : (e₁ * e₂) x = e₁ (e₂ x) := rfl
+
+/-- An algebra isomorphism induces a group isomorphism between automorphism groups -/
+def aut_congr (ϕ : A₁ ≃ₐ[R] A₂) : (A₁ ≃ₐ[R] A₁) ≃* (A₂ ≃ₐ[R] A₂) :=
+{ to_fun := λ ψ, ϕ.symm.trans (ψ.trans ϕ),
+  inv_fun := λ ψ, ϕ.trans (ψ.trans ϕ.symm),
+  left_inv := λ ψ, by { ext, simp_rw [trans_apply, symm_apply_apply] },
+  right_inv := λ ψ, by { ext, simp_rw [trans_apply, apply_symm_apply] },
+  map_mul' := λ ψ χ, by { ext, simp only [mul_apply, trans_apply, symm_apply_apply] } }
+
```

Comment:
Could you add the following lemmas? We have these for lots of other `congr`-style equiv lemmas:

```suggestion
@[simp] lemma aut_congr_refl : aut_congr (alg_equiv.refl _) = mul_equiv.refl := sorry
@[simp] lemma aut_congr_symm : (aut_congr ϕ).symm = aut_congr ϕ.symm := sorry
@[simp] lemma aut_congr_symm : (aut_congr ϕ).trans (aut_congr ϕ') = aut_congr (ϕ.trans ϕ') := sorry
```